readme.md

```diff
@@ -215,16 +215,18 @@ Type1|Type2               // Union
 
 > [See Docs](https://developers.google.com/closure/templates/docs/commands#switch "Switch")
 
-###### elvis
+###### ternary
 
 ```
-Hello {$name ?: 'there'}
+{isNonnull($name) ? 'Hello {$name}' : 'Hello there'}
```

Comment:
Thanks a lot @yuchi! Everything looks good except one small detail. I'm trying to progressively explain features and increase complexity as people scroll down on this file. That means I try not to reference something from they haven't seen before.

So there are two things we can do here:
1. Not use `isNonnull` since it's only showed on the bottom of the document.
2. Move the `Functions` section before the `Conditionals` section.

My preference is solution number 1, what about you?